extensions/scarb-snforge-test-collector/src/compilation/test_collector/config.rs

```diff
@@ -140,6 +145,23 @@ pub fn forge_try_extract_test_config(
         None
     };
 
+    let max_steps = if let Some(attr) = max_steps_attr {
+        if attr.args.is_empty() {
+            None
+        } else {
+            extract_max_steps(attr, db).on_none(|| {
+                diagnostics.push(PluginDiagnostic {
+                    severity: Severity::Error,
+                    stable_ptr: attr.args_stable_ptr.untyped(),
+                    message: "Expected max steps must be <u32>"
```

Comment:
Doesn't align exactly with the other error messages

Reply:
@piotmag769 updated let me know what do you think?

Reply:
I was thinking of "Expected max steps config must be of the form `max_steps: <u32>`"

Reply:
Ok, but imo it sound like `#[max_steps({max_steps: 3232})]`

Reply:
Up to you